### ISSUE TITLE
The count of the results thrown in the Add Nuget Package window, are not being announce by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#268]
### Steps to Reproduce

1. Launch the Xamarin Workbooks, and Turn ON the Voiceover AT.
2. "Create a new Workbook" window-> Go to Mac tab which is just above the "Create Workbook" button.
3. Select Xamarin.Mac (Modern)-> Create Workbook-> Hit on Enter.
4. Hit on "Command+G" keys to add the Nuget Package through Nuget Package Manager window. 
5. Search the Nuget package in the search edit of the package window. 

### Expected Behaviour
While Voiceover user is searching the Nuget package through the package manager window, the Voiceover should announce the count of there results obtained.

### Actual Behaviour
The count of the results obtained in Add Nuget Package  window are not being announce by the Voiceover.
Note: Voiceover focus is traversing the package results table but its not announcing the count.
Its also not announcing the position and size of set, while Voiceover focus is on the individual thrown results.

### User Impact:
The Voiceover user won’t be able to know the counts of the results obtained in the add nuget package window.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)

### Log File

[VO3_AddPackageResults.zip](https://github.com/Microsoft/workbooks/files/1990841/VO3_AddPackageResults.zip)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1824
